activation/activation.go

```diff
@@ -562,9 +559,8 @@ func (b *Builder) PublishActivationTx(ctx context.Context) error {
 		logger.With().Info("using existing atx challenge", log.Stringer("current_epoch", b.currentEpoch()))
 	} else {
 		logger.With().Info("building new atx challenge", log.Stringer("current_epoch", b.currentEpoch()))
-		err := b.buildNIPostChallenge(ctx)
-		if err != nil {
-			return fmt.Errorf("failed to build new atx challenge: %w", err)
+		if err := b.buildNIPostChallenge(ctx); err != nil {
+			return fmt.Errorf("build new atx challenge: %w", err)
```

Comment:
👍 I'm generally for removing "failed/error" keywords from `fmt.Errorf(...)`. What do you think about using present continuous time in wrappers instead? I think it would build nicer log messages.

```suggestion
			return fmt.Errorf("building new atx challenge: %w", err)
```

Reply:
i am not sensitive to language like this. for me the purpose of this text is just something i can grep the log with.
my main goal is actually to make it short/concise as possible. here i don't see a big difference.

Reply:
It's not that big a difference. But consider:
> "Failed to publish ATX", err = "building new tax challenge: getting positioning ATX: not found"
vs
> "Failed to publish ATX", err = "build new tax challenge: get positioning ATX: not found"

The former describes what it _was doing_ when the error occurred.

Anyway. I'm fine with either option.

Reply:
i understand the difference in the english language, i just thought that it is not strictly necessary to be grammatically correct with the log msgs.

my criteria is the shortest msg that convey the meaning. in this case i could easily do it, except  i will have to do this in many 0laces to be consistent.

i wont object if you prepare a PR for that.